README.md

```diff
@@ -168,3 +168,7 @@ See more:
 * [Security Policy](SECURITY.md)
 * [Security Advisories](/security/advisories)
 
+## Note
```

Comment:
We have `Notice` point above, do we want to have additional `Note` here?

Reply:
I move the note as the 5th item in Notice section